markhor_bringup/package.xml

```diff
@@ -12,8 +12,12 @@
   
   <build_depend>visp_auto_tracker</build_depend>
 
+  <build_depend>joy</build_depend>
```

Comment:
Très bonne initiative ! :+1:

Reply:
Is there any other changes ?

Reply:
Yes, there's the file name : "c" that should not be there. But that should be it.